src/app.js

```diff
@@ -165,6 +166,9 @@ router.use((req, res, next) => {
 router.use("/oauth2", require("./app/oauth2/router"));
 router.use("/credential-issuer", require("./app/credential-issuer/router"));
 router.use("/ipv", require("./app/ipv/router"));
+if (ENABLE_PREVIEW) {
+  router.use("/dev", require("./app/development/router"));
```

Comment:
This change of URL may be controversial as we'll need to let the UX team know and they'll have to update bookmarks. We could revert this if required.

Reply:
Might be best if we can avoid this - It will affect support folk that have been using this too
and I guess it will break the journey map links too?